resources/wikia/modules/tracker.js

```diff
@@ -267,11 +267,14 @@
 				}
 			}
 
+			callbacks = data.callbacks || callbacks;
 			browserEvent = data.browserEvent || browserEvent;
 			eventName = data.eventName || eventName;
 			trackingMethod = data.trackingMethod || trackingMethod;
 			tracking[ trackingMethod ] = true;
 
+			delete data.callbacks;
+
 			if ( tracking.none || ( tracking.analytics &&
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1067)